utils/nlp.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+import spacy
+import nltk
+from nltk.corpus import stopwords
+
+
+def text_preprocess(text: str, lang: str = "english") -> List[str]:
+    """
+    Does some preprocessing to better analyze a text.
+    Lowercase, remove punctuation and stopwords, tokenization and lemmatization
+
+    Returns:
+    List of processed tokens
+    """
+    if lang == "english":
+        nlp = spacy.load("en_core_web_md")
+    elif lang == "french":
+        nlp = spacy.load("fr_core_news_md")
+    else:
+        raise NotImplementedError()
+    
+    word_tokens = nltk.word_tokenize(text)
```

Comment:
Refactor the code to avoid unnecessary assignment since the variable is reassigned in line 29.
```suggestion
```